root/etc/cont-init.d/98-golang

```diff
@@ -4,7 +4,10 @@ echo "ensuring golang is in PATH"
 if grep -q -E '^(export )?PATH=' /etc/services.d/code-server/run; then
   if ! grep -q -E '^(export )?PATH=.*/usr/local/go/bin.*' /etc/services.d/code-server/run; then
     sed -i '/PATH/ s/$/:\/usr\/local\/go\/bin/' /etc/services.d/code-server/run
+  elif ! grep -q -E '^(export )?PATH=.*/config/go/bin:.*' /etc/services.d/code-server/run; then
+    sed -i 's/PATH=/PATH=\/config\/go\/bin:/g' /etc/services.d/code-server/run
+  fi
   fi
 else
-  sed -i '/^#!\/usr\/bin/a \\n# Added by codeserver-golang\nexport PATH=$PATH:/usr/local/go/bin' /etc/services.d/code-server/run
+  sed -i '/^#!\/usr\/bin/a \\n# Added by codeserver-golang\nexport PATH=/config/go/bin:$PATH:/usr/local/go/bin' /etc/services.d/code-server/run
```

Comment:
Can you move `config/go/bin` to the end, next to `/usr/local/go/bin`? It's triggering my OCD :)

Reply:
I was going to do that, but then considered the case where a user wanted their binaries first in the path. This does that.

Reply:
ok, that makes sense